### PR TITLE
Update resources to deal with resource not found case according to terraform feedback

### PR DESCRIPTION
**Pull request by zhiqiangjin**
If a resource cannot be found(404) in the read function, do a d.SetId("") to remove it from the state file. No need to return an error.

### CODE REVIEW

File: goaviatrix/aws_tgw_vpc_attachment.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func (c *Client) GetAwsTgwVpcAttachment(awsTgwVpcAttachment *AwsTgwVpcAttachment
 	aTVA, err := c.GetAwsTgwDomainAttachedVpc(awsTgwVpcAttachment)
```
Comment (arvindsree) [inline goaviatrix/aws_tgw_vpc_attachment.go]: is this API used by tgw as well?

File: goaviatrix/license.go [hunk 2]
```diff
@@ -92,5 +92,8 @@ func (c *Client) GetCustomerID() (string, error) {
 	if err = json.Unmarshal(body, &response); err != nil {
 		return "", errors.New("Json Decode list_customer_id failed: " + err.Error())
 	}
+	if response.Results == "" {
```
Comment (arvindsree) [inline goaviatrix/license.go]: this just means customer ID is empty. It isn't a not found

File: goaviatrix/aws_tgw_vpc_attachment.go (hunk 1 above)
Comment (zhiqiangjin) [reply goaviatrix/aws_tgw_vpc_attachment.go]: It is only used in resource_aws_tgw_vpc_attachment.

File: goaviatrix/license.go (hunk 2 above)
Comment (zhiqiangjin) [reply goaviatrix/license.go]: Is not customer_id a non-empty string? If yes, getting an empty string means not found.

**Pull request merged**